learn/templates/learn/_high_level_progress.html

```diff
@@ -5,6 +5,6 @@
     {{module_completion_data.completion_count}}
     /
     {{module_completion_data.total_pages}}
-    lessons complete</p>
+    lesson{% if module_completion_data.total_pages > 1 %}s{% endif %} completed</p>
```

Comment:
Let's use `pluralize` for consistency

```suggestion
    lesson{{ module_completion_data.total_pages|pluralize}} completed</p>
```